components/places/android/src/main/java/org/mozilla/places/PlacesConnection.kt

```diff
@@ -256,14 +337,43 @@ class SyncAuthInfo (
 )
 
 /**
- * An API for interacting with Places.
+ * An API for interacting with Places. This is the top-level entry-point, and
+ * exposes functions which return lower-level objects with the core
+ * functionality.
  */
-interface PlacesAPI {
+interface PlacesApiInterface {
```

Comment:
Thom pointed this out earlier, but it's worth re-flagging for @grigoryk - is there a better naming convention for these interfaces (and more to the point, are they even required?)

Reply:
I don't think we'll hear back on this soon, since Grisha is currently in Russia IIUC.